src/applications/sco/tests/accessibility.cypress.spec.js

```diff
@@ -103,7 +103,7 @@ describe('Accessibility', () => {
     // Should be focused on right panel now
     cy.focused().should('contain.text', 'Access Enrollment Manager');
     cy.realPress('Tab');
-    cy.focused().should('contain.text', 'Expand all +');
```

Comment:
The fix for this [card](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2524) changed the way the expand all/collapse all button is rendered when there are expanded items on the page. The change to this tests reflects that change.